rllib/agents/a3c/a3c_torch_policy.py

```diff
@@ -68,9 +71,10 @@ def torch_optimizer(policy, config):
 
 class ValueNetworkMixin:
     def _value(self, obs):
-        obs = torch.from_numpy(obs).float().unsqueeze(0).to(self.device)
-        _ = self.model({"obs": obs}, [], [1])
-        return self.model.value_function().detach().cpu().numpy().squeeze()
+        with torch.no_grad():
```

Comment:
Just to double check that there's never a time this is called when you might want gradients such as when computing a vf_loss and need gradients to update the value network?

Reply:
See my comment below on `self._value` (used only for fetching information, never in the loss).

Reply:
I think we can omit this no grad thing if it's always inserted at a higher level in the policy template right?